pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -372,6 +373,7 @@ func (qjm *XController) PreemptQueueJobs(inspectAw *arbv1.AppWrapper) {
 				qjm.qjqueue.AddUnschedulableIfNotPresent(updateNewJob)
 
 			}
+			generatedCondition = true
```

Comment:
Shouldn't this be inside the if statement in 351? As in only preempt if the job did overrun?

Reply:
Thanks, Done